### PR TITLE
Db style tests

### PR DESCRIPTION
**Pull request by BryanGK**
This PR is for adding the database style tests submodule and adding them to the github workflow

### CODE REVIEW

**Comment by BryanGK** [comment]
Need to figure out how to run submodule tests on CI

**Comment by BryanGK** [comment]
Need to figure out how to run submodule tests on CI

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -79,5 +79,7 @@ jobs:
           make
           make install
           psql -c 'CREATE EXTENSION pgtap;' || true
-      - name: Run Tests
-        run: pg_prove --username ${{ env.PGUSER }} --dbname ${{ env.PGDATABASE }} db/test/*/*_test.sql
+      - name: Run Database Unit Tests
+        run: pg_prove --username ${{ env.PGUSER }} --dbname ${{ env.PGDATABASE }} db/test/unit/**/*_test.sql
+      - name: Run Database Style Tests
+        run: pg_prove --username ${{ env.PGUSER }} --dbname ${{ env.PGDATABASE }} db/test/style/*_test.sql --set schemas_to_test=ccbc_public,ccbc_private
```
Comment (wenzowski) [inline .github/workflows/tests.yaml:85]: the `db/test/style` folder is coming in via `.gitmodules` added in this PR but it looks like the checkout action isn't pulling in the git submodule. Perhaps an update to the checkout step?
```yaml
steps:
- name: Checkout repository and submodules
  uses: actions/checkout@v2
  with:
    submodules: recursive
```

Comment (marcellmueller) [reply .github/workflows/tests.yaml:87]: Thanks, this worked. Also I had to properly install the module with `git submodule add https://github.com/cas.... /db/test/style`.

**Review by wenzowski — APPROVED** [review]

**Pull request merged**